### PR TITLE
[WIP] Implementing oAuth2

### PR DESCRIPTION
**Pull request by nicosomb**
* remove wsse implementation 
* add oAuth2 implementation

Do not merge.

### CODE REVIEW

**Comment by nicosomb** [comment]
@j0k3r I followed this tutorial http://blog.tankist.de/blog/2013/07/17/oauth2-explained-part-2-setting-up-oauth2-with-symfony2-using-fosoauthserverbundle/, I've got a problem I explained here: http://blog.tankist.de/blog/2013/07/18/oauth2-explained-part-3-using-oauth2-with-your-bare-hands/#comment-2176424716

File: app/config/config.yml [hunk 1]
```diff
@@ -137,7 +137,7 @@ liip_theme:
     themes:
         - baggy
         - material
-    autodetect_theme: wallabag_core.helper.detect_active_theme
+    # autodetect_theme: wallabag_core.helper.detect_active_theme
```
Comment (j0k3r) [inline app/config/config.yml]: Why?

Comment (nicosomb) [inline app/config/config.yml]: just a test, because wallabag tried to load the active theme when I did some tests. I will enable it soon.

**Comment by nicosomb** [comment]
About my above problem, I think I found the solution in the train. I'll try today.

**Comment by nicosomb** [comment]
I prefer close this PR, FosUser was added in an other PR, it will ne easier for me to have a clean branch.

**Pull request closed without merging**